### DISCUSSION TITLE
Any way to use Http instead of Https?

### DISCUSSION THREAD

**Discussion by DesBaril** [mikaelvesavuori/figmagic#171] [Q&A] [answered]
Hi, 
I've implemented figmagic in my project, but I get a self-signed certificate error when trying to sync with the figma source.
So i'm trying to find a way to use figmagic with the http protocol instead of https.
Any idea?
thanks in advance!

**Comment by mikaelvesavuori** [comment, marked as answer]
Hey @DesBaril !

This will sound really dumb but I never got informed something happened in the GitHub Discussions... Sorry about that!

I am not planning on implementing that change as such, as it would require adding additional logic around the use of Node's `http` or `https` modules. It's quite messy already.

However, quite some time ago we had a similar-ish kind of situation around using custom certs; see https://github.com/mikaelvesavuori/figmagic/commit/ef572b712ef2f9ec3499a41b478e4b2a3fccfa08.

Is that something you could use to solve your issue?
